<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGame</class>
 <widget class="QDialog" name="NewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>-120</y>
     <width>141</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="addGameLabel">
      <property name="text">
       <string>Добавити матч</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ConfirmNewGameBtn">
      <property name="text">
       <string>Добавити</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>221</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="team1LE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="team2LE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="timeLE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="stadiumLE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="sectorLE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="rowLE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="placeLE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="team1Label">
        <property name="text">
         <string>Клуб 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="team2Label">
        <property name="text">
         <string>Клуб 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>Час</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stadiumLabel">
        <property name="text">
         <string>Стадіон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sectorLabel">
        <property name="text">
         <string>Сектори</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rowLabel">
        <property name="text">
         <string>Ряди</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="placeLabel">
        <property name="text">
         <string>Місця</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
